whats_new/deprecated-features.adoc

```diff
@@ -27,14 +27,13 @@ now deprecated.
 |Replaced by Prometheus adapter.
 
 |Atomic Host
-|Replaced by Red Hat Enterprise Linux CoreOS.
+|Replaced by Fedora CoreOS.
```

Comment:
`{op-system-first}` is `Fedora CoreOS (FCOS)` or `Red Hat Enterprise Linux CoreOS (RHCOS)` - I guess this one would work better for OKD and OCP docs?

Reply:
Good catch. This file is only in OKD so I went with the Fedora over the variable. But I changed it to the variable it for consistency.